includes/rest-api.php

```diff
@@ -170,18 +165,13 @@ function receive_variations( \WP_REST_Request $request ) {
 
 	if ( \DT\NbAddon\WC\Utils\is_assoc( $variation_data ) ) {
 		\DT\NbAddon\WC\Utils\sync_variations( $post_id, $variation_data['current_variations'] );
-		$res = \DT\NbAddon\WC\Utils\set_variation_update( $variation_data, $post_id );
+		$result = \DT\NbAddon\WC\Utils\set_variation_update( $variation_data, $post_id );
 	} else {
 		\DT\NbAddon\WC\Utils\sync_variations( $post_id, $variation_data[0]['current_variations'] );
-		$res = \DT\NbAddon\WC\Utils\set_variations_update( $variation_data, $post_id );
+		$result = \DT\NbAddon\WC\Utils\set_variations_update( $variation_data, $post_id );
 	}
-	/**
-	 * Action triggered after variations update in spoke
-	 *
-	 * @param int $post_id Parent post ID.
-	 */
-	do_action( 'dt_variations_updated', $post_id, $request );
-	return $res;
+
+	return apply_filters( 'dt_variations_updated', $result, $post_id, $request );
```

Comment:
@armdev17 better to have annotations if possible.

Reply:
@arsendovlatyan added in https://github.com/NovemBit/distributor-wc-addon/commit/5c064ab23e1c89c117516617c7e7becbb9ba150f.